### ISSUE TITLE
Replace `proc-macro-error`

### ISSUE THREAD

**Issue by max-sixty** [binary-banter/test-each-file#4]
Currently the only dependency that use `syn` v1 in PRQL's dependencies is from `test-each-file`. Given how large `syn` is, this slows down our compile times.

I think that's from `proc-macro-error`, which isn't maintained:

```sh
$ cargo tree -i syn@1

syn v1.0.109
└── proc-macro-error v1.0.4
    └── test_each_file v0.3.1 (proc-macro)
```

I think there are some replacements for that crate — to the extent it's possible, could that it be replaced?

**Comment by JonathanBrouwer** [comment]
That's a good point. I'm going to take a look at how feasible replacing this dependency is tonight

**Comment by Vlamonster** [comment]
Issue is resolved in version 0.3.2.

**Comment by max-sixty** [comment]
Thanks @Vlamonster !
